WordPress/WordPressHomeWidgetToday/View/TodayWidgetView.swift

```diff
@@ -0,0 +1,63 @@
+import SwiftUI
+import WidgetKit
+
+struct TodayWidgetView: View {
+    let content: TodayWidgetContent
+
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading) {
+                Text(content.siteTitle)
+                    .font(.footnote)
+                    .fontWeight(.semibold)
+                    .lineLimit(Constants.siteTitleLines)
+                    .foregroundColor(Color(.label))
+
+                Text(Constants.todayTitle)
+                    .font(.caption)
+                    .fontWeight(.regular)
+                    .foregroundColor(Color(.secondaryLabel))
+
+                Spacer()
+
+                Text(Constants.viewsTitle)
+                    .font(.caption)
+                    .fontWeight(.semibold)
+                    .foregroundColor(Constants.viewsTitleColor)
+
+                Text("\(content.views)")
```

Comment:
I'm impressed that SwiftUI is able to auto-localize this and that formatting of numbers work transparently there 👌 
_(I'm guessing because `Text` takes a `LocalizedStringKey` and this type has a custom `StringInterpolation` overload for number types which applies formatting to it, 🤯 )_

I've tested that on a simulator set to French and it's properly formatted with a thin space – as expected in French locale – instead of a comma like English does, so 👍